src/components/staking/SavingsTab.vue

```diff
@@ -51,8 +51,8 @@ export default defineComponent({
             void store.dispatch('account/resetTransaction');
             if (
                 toSavingAmount.value === '0.0000' ||
```

Comment:
This comparison should be reviewed. It would not work with another chain that has different precision and I would like for us to use just `0`.

Reply:
I agree. Created [this issue](https://github.com/telosnetwork/open-block-explorer/issues/643) for future planification